astropy/coordinates/tests/test_solar_system.py

```diff
@@ -207,7 +208,8 @@ def setup(self):
     def test_erfa_planet(self, body, sep_tol, dist_tol):
```

Comment:
Oops - good spot @mhvk. This is in the wrong place! I didn't intend to add Kitt Peak tests for the Moon, as these depend on #4986 being merged to pass...